frontend/components/header.js

```diff
@@ -2,7 +2,9 @@ import React from 'react';
 
 export default () => (
   <section>
-    <img alt="Shields.io" src="static/logo.svg" />
+    <a href="/">
```

Comment:
I think making this `/#/` will avoid the redirect that's occurring after clicking it.